.compose/traefik/dynamic/dev.yml

```
@@ -0,0 +1,107 @@
+
+  services:
+    tom-local:
+      loadBalancer:
+        servers:
+          - url: "http://host.docker.internal:3000"
+    
+    fed-local:
+      loadBalancer:
+        servers:
+          - url: "http://host.docker.internal:3001"
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)